docs/installation_guide.md

```diff
@@ -302,17 +320,19 @@ Replace the first "80" to a customized port, e.g. 8888:80.
 
```

Comment:
As for the section "Configuring Harbor listening on a customized port" it looks like it should be part of FAQ rather than in installation guide?  could you check the faq and see if there's guide on how to customize the port, and add a link to it in this guide?

Reply:
No FAQ found in the docs